gwpy/timeseries/timeseries.py

```diff
@@ -1840,7 +1840,7 @@ def q_transform(self,
                     frange=qtransform.DEFAULT_FRANGE,
                     gps=None,
                     search=.5,
-                    tres=.001,
+                    tres="<default>",
```

Comment:
Please add documentation of the defaults in the docstring, similar to what is there for `fres`.

Reply:
@duncanmmacleod, this is done in the latest commit. I also updated the documentation of `outseg` to give its default (the full duration of the input `TimeSeries`).